database_models.go

```diff
@@ -160,9 +161,49 @@ const (
 // the file name and which build it belongs to.
 type Artifact struct {
 	ArtifactID uint   `gorm:"primaryKey" json:"artifactId"`
-	BuildID    uint   `gorm:"not null;index:param_idx_build_id" json:"buildId"`
+	BuildID    uint   `gorm:"not null;index:artifact_idx_build_id" json:"buildId"`
```

Comment:
Don't know how this has worked before, how GORM has resolved this name conflict, but if GORM had set up the index named `param_idx_build_id` then I think you need to add a migration step to remove the said index from the `artifacts` table. I've not double-checked, but is the `param_idx_build_id` index only applied to one of the tables or both, and will this change make any duplicates or will it resolve itself by GORM's automatic migrations?